### PR TITLE
Limit bash scripting to task 1

### PR DESCRIPTION
**Pull request by SashaWeinstein**
Only ask for bash scripting as a bonus item for task 1. The data challenges we got for the data engineering position in the summer/fall of 2022 had a lot of copy/pasted bash code. Challenge will faster to complete if we only ask for it once. I got hung up on how to describe the date filter. I'm not sure if this language
>> Write a python script to pulls data from the NYC Open DataAPI based two filters. The first filter is on responding agency. The second filter is an integer date filter to only get calls `n` days before the current date.

is sufficiently clear

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,13 +44,17 @@ Your code interview will be evaluated based on your repo, so make sure all files
...
 
 ### Python Task 1: Data Download
 
-To download 311 service request records write a proccess that 2 parameters passed from the command line and pulls the data from the API. The process shoul start with a bash script that parses the parameters of number of days and responding agency acronym then calls python code to hit the API. For example, if a user wanted to get all service request records created in the last week where DSNY is the responding agency, they would pass `7` and `DSNY` as the parameters. For this exercise, we ask that you download all 311 service requests filed the **last seven days** where **HPD** is the responding agency.  Save the data as a csv named `raw.csv` in a folder called `data`. 
+First you need to download 311 service request records. Write a python script to pulls data from the NYC Open DataAPI based two filters. The first filter is on responding agency. The second filter is an integer date filter to only get calls `n` days before the current date. The script should download the associated service requests to python memory and cache to a .csv.
```
Comment (mbh329) [inline README.md:51]: For this exercise, you will be working with the NYC 311 service request data. To do this, you will first need to write a python script to pull data from the NYC OpenDataAPI (**Do we want to mention socrata**) with two filters: a filter for the responding agency and a integer filter corresponding to the number of days before the current date (to get only calls within a certain day threshold i.e. the last 7 days). The script should download the associated service requests to python memory and cache to a .csv.

Comment (SashaWeinstein) [reply README.md:51]: I took some of your language on how to describe the filters

File: README.md [hunk 2]
```diff
@@ -44,13 +44,17 @@ Your code interview will be evaluated based on your repo, so make sure all files
...
 
-*Bonus points* if you demostrate your code is dynamic by downloading and saving a series of csv files each with a different timeframe and responding agency, and programmatically save these files as csvs with a naming convention of your choice.
+For example, if a user wanted to get all service request records created in the last week where DSNY is the responding agency, they would pass `7` and `DSNY` as the parameters. 
+
```
[the remark this answers is not in the document]
Comment (SashaWeinstein) [reply README.md:54]: I wanted to put the example closer to the overview of the task as I don't want the reader to be confused about what the task is overall. I think having the exact instructions at the bottom is a common convention

**Review by mbh329 — APPROVED** [review]
LGTM 🥇

File: README.md [hunk 3]
```diff
@@ -44,13 +44,17 @@ Your code interview will be evaluated based on your repo, so make sure all files
 
 We love the NYC 311 service and the open data products that come with it. In this challenge, you will use **[311 Service Requests from 2010 to Present](https://data.cityofnewyork.us/Social-Services/311-Service-Requests-from-2010-to-Present/erm2-nwe9)** on NYC Open Data, write an ETL pipeline, and produce some data insight.
 
-This challenge has two halves, a python/bash half with 4 steps and a docker/SQL part with 2 steps. The second half will use a file generated by the first python step. It's our expectation that the logic to perform each step in the first half of this challenge will be written in python, and be callable by an executable script that takes specified arguments as inputs.
+This challenge has two halves, a python half with 4 steps and a docker/SQL part with 2 steps. The second half will use a file generated by the first python step. It's our expectation that the logic to perform each step in the first half of this challenge will be written in python.
```
[the remark this answers is not in the document]
Comment (SashaWeinstein) [reply README.md]: Added a commit to address

**Comment by mbh329** [comment]
Great, thanks. I prematurely approved but I think as is it looks great so I stand by my approval

**Comment by SashaWeinstein** [comment]
Sick, thanks for reviewing

**Pull request merged**